app/sequencescape_excel/sequencescape_excel/specialised_field/primer_panel.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SequencescapeExcel
+  module SpecialisedField
+    ##
+    # The library type is a value which must already exist.
+    # Weirdly the library type is stored as a value rather than an association.
```

Comment:
Should these comments go? Copy-paste remnants?

Reply:
Whoops. Yes. Bad James.

Reply:
Fixed